docs/data/material/discover-more/showcase/showcase.md

```diff
@@ -1,13 +1,13 @@
 # Showcase
 
-<p class="description">The following is a list of some of the public apps using MUI.</p>
+<p class="description">Check out these public apps using MUI's products to get inspired for your next project.</p>
 
-Are you using MUI? [Share your product](https://github.com/mui/material-ui/issues/22426).
+Are you working with MUI's component libraries? [Show us what you're building](https://github.com/mui/material-ui/issues/22426)! We'd love to see it.
 
-The websites we include rank well in at least one of these dimensions:
+To be featured on this page, your app should meet the following requirements:
```

Comment:
Just to make sure: do I have to meet **all of them** or **some of them**?

Reply:
That is a good question. I would think that at least one of the criteria would be sufficient.

Reply:
Updated to make that more clear.

Reply:
At least one 👍